flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -73,7 +75,9 @@
   // TableLoader to load iceberg table lazily.
   private final TableLoader tableLoader;
   private final SerializableConfiguration hadoopConf;
+  private final org.apache.flink.configuration.Configuration flinkConf;
   private final boolean replacePartitions;
+  private final long flushCommitInterval;
```

Comment:
It sounds a confusing name ?  For StreamingFileSink, we use the rollingCheckInterval. 

```java
	public static final ConfigOption<Duration> SINK_ROLLING_POLICY_CHECK_INTERVAL = key("sink.rolling-policy.check-interval")
			.durationType()
			.defaultValue(Duration.ofMinutes(1))
			.withDescription("The interval for checking time based rolling policies. " +
					"This controls the frequency to check whether a part file should rollover based on 'sink.rolling-policy.rollover-interval'.");
```

Reply:
Yes, here I think it's better to use commitInterval rather than flushCommitInterval.